app/models/manageiq/providers/redhat/infra_manager/event_catcher/runner.rb

```diff
@@ -1,4 +1,4 @@
-class ManageIQ::Providers::Redhat::InfraManager::EventCatcher::Runner < ManageIQ::Providers::BaseManager::EventCatcher::Runner
+class ManageIQ::Providers::Redhat::InfraManager::EventCatcher::Runner < ManageIQ::Providers::Ovirt4::InfraManager::EventCatcher::Runner
   def event_monitor_handle
     require 'ovirt_provider/events/ovirt_event_monitor'
     @event_monitor_handle ||= OvirtEventMonitor.new(event_monitor_options)
```

Comment:
wouldnt it be enough to have a `OvirtEventMonitor` for version 4? Because thats basically the the abstraction layer.
Both would have the same api (like `start` and `each_batch`)